api/controllers/HeartbeatController.js

```diff
@@ -13,7 +13,7 @@ const formatMaxMinMean = (collection, key) => {
 module.exports = {
   _config: {
     populate: true,
-    limit: 0
+    limit: 9223372036854775807
   },
```

Comment:
if a limit is specified here it will override how the query is formatted when it sent to the adapter. same as specifying populate and anything else we shove in the _config of a controller that is reflecting a model